main.go

```diff
@@ -384,9 +384,15 @@ func (o *Options) runServerMode() error {
 		o.DBOnlyMode,
 	)
 
+	// Initial metrics refresh to get the endpoint scrapable ASAP and prevent prom gaps, before
+	// we start the lengthy mat view refreshes.
+	if err := server.RefreshMetricsDB(); err != nil {
```

Comment:
Do we need to check dbOnlyMode here or do we see that mode going away? that check is in the regular server.refreshMetrics call